### PR TITLE
assignment-1 has been complete

### PR DESCRIPTION
**Pull request by ekveliasvili**
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
we used Python and different libraries to build a model 
## What did you learn from the changes you have made?
learned how to use KNN models, split data for train and test sets and calculate the accuracy of the model we build
## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
I was trying to use train_test_split function but could not figure the correct syntax, so with the help of LS we came up with another way of splitting the data
## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
yes, I got stumbled several times when completing the assignment
## How were these changes tested?
i ran code to test it
## Checklist
- [ x] I can confirm that my changes are working as intended

### CODE REVIEW

**Review by KasraVakiloroayaei — APPROVED** [review]
A1: Complete 

Q2part(ii) since it is a categorical variable, we do not standardize it 
Q3 - When you use parameter grid, the third argument for the range function, is to skip x values. Thus not training on the entire data frame, thus not finding the most optimal k for KNN. Try removing the third argument, then you will notice the value for best k will change. 
Q4 - Please calculate the accuracy once again after making changes finding a new best n-neighbor

**Comment by ekveliasvili** [comment]
Thank you! I re ran the code without the skip number and got the best number of neighbors at 8. But the accuracy did not change
